src/Features/CSharp/Portable/RemoveUnnecessaryLambdaExpression/CSharpRemoveUnnecessaryLambdaExpressionDiagnosticAnalyzer.cs

```diff
@@ -0,0 +1,273 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I think this should be in `Analyzers` not `Features`. The link to this comment doesn't work, so here is a screenshot:

![image](https://user-images.githubusercontent.com/31348972/149610904-3a864f15-bbef-41cc-b109-503f26218e35.png)

Reply:
@CyrusNajmabadi @mavasani There are few analyzers in `Features` layer. Do we need an issue to move them?

Reply:
moved.  THanks!

Reply:
> Do we need an issue to move them?

i'm not a fan of issues for that.  might as well just do the work :)